File: Selenium/Project/Activity2.java
import org.openqa.selenium.By;
import org.openqa.selenium.WebElement;
import org.testng.annotations.Test;

public class Activity2 extends BaseClass {
    @Test
    public void activity2() {
        WebElement heading = driver.findElement(By.xpath("//h1[text()=' Learn from Industry Experts ']"));
        if (heading.getText().equals("Learn from Industry Experts")) {
        }
    }
}

File: Selenium/Project/Activity2.py
from selenium import webdriver
from selenium.webdriver.common.by import By

# Create an instance of the Firefox webdriver
driver = webdriver.Firefox()

# Open the webpage
driver.get("https://alchemy.hguy.co/lms")

# Find the heading element using XPath
heading = driver.find_element(By.XPATH, "//h1[text()=' Learn from Industry Experts ']")

# Check if the heading text is 'Learn from Industry Experts'
if heading.text == "Learn from Industry Experts":
    pass

# Close the browser
driver.quit()
